packages/kbn-docs-utils/src/api_docs/build_api_declarations/extract_import_refs.ts

```diff
@@ -115,7 +115,7 @@ function getScopeFromPath(path: string, plugin: KibanaPlatformPlugin, log: Tooli
   } else if (path.startsWith(`${plugin.directory}/common/`)) {
     return ApiScope.COMMON;
   } else {
-    log.warning(`Unexpected path encountered ${path}`);
+    log.warning(`Unexpected path encountered ${path} for plugin directory ${plugin.directory}`);
```

Comment:
Adding extra information because I'm getting some weird stuff from here, like, 

> Unexpected path encountered /Users/gammon/Elastic/kibana2/x-pack/plugins/security_solution/server/client/client for plugin directory /Users/gammon/Elastic/kibana2/x-pack/plugins/security

I'll need to dig deeper into why this is happening.

Reply:
Fixed this bug